### PR TITLE
add a async test with axios call using Jest and fixed bugs with rendering the default style

### PR DESCRIPTION
**Pull request by quanjingchen**
in order to use async/await, I installed a new package:     "regenerator-runtime": "^0.13.11",

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "morgan": "^1.10.0",
     "nodemon": "^2.0.20",
     "react-test-renderer": "^18.2.0",
+    "regenerator-runtime": "^0.13.11",
```
Comment (kevinleaves) [inline package.json:45]: curious to see what this does. if it makes your tests work, i'm all for it.

File: spec/__overviewTests__/ImageGallery.test.js [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+      baseURL,
+      url,
+      headers: {
+        Authorization: process.env.API_KEY,
+      },
+    };
+    const response = await axios(options);
+    const style = response.data.results[0];
+    render(<ImageGallery currentStyle={style} />);
+    expect(await screen.getAllByRole('img', { name: 'thumbnail' }).length).toBe(7);
```
Comment (kevinleaves) [inline spec/__overviewTests__/ImageGallery.test.js:28]: this looks really good. can't wait to use this technique to implement more tests in my components.

**Review by kevinleaves — APPROVED** [review]
Good tests. Seems like you got everything to work. Approved.

File: spec/__overviewTests__/ImageGallery.test.js (hunk 2 above)
Comment (wooseok0717) [reply spec/__overviewTests__/ImageGallery.test.js:28]: +1 Awesome Job!

**Pull request merged**